samples/ComputerVision/AreaOfInterest/Program.cs

```diff
@@ -41,12 +41,12 @@ public static async Task RunAsync(string endpoint, string key)
             string remoteImageUrl = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/faces.jpg";
 
             Console.WriteLine("area of interest being found ...");
-            await RoiFromUrlAsync(computerVision, remoteImageUrl);
-            await RoiFromStreamAsync(computerVision, localImagePath);
+            await AreaOfInterestFromUrlAsync(computerVision, remoteImageUrl);
+            await AreaOfInterestFromStreamAsync(computerVision, localImagePath);
         }
 
         // Analyze a remote image
-        private static async Task RoiFromUrlAsync(ComputerVisionClient computerVision, string imageUrl)
+        private static async Task AreaOfInterestFromUrlAsync(ComputerVisionClient computerVision, string imageUrl)
```

Comment:
Let's make it consistent with the API name, and call it GetAreaOfInterestFromUrlAsync